platforms/hyperledger-indy/configuration/roles/create/helm_component/ledger_txn/tasks/nested_main.yaml

```diff
@@ -0,0 +1,69 @@
+################################################################################################
```

Comment:
You use component_name as the name for both organisations (admin org and endorser org), these can be different orgs. Same with org_vault_url...

Reply:
Changed the names, now different for both admin and new identities